### PR TITLE
Remove prerelease from curriculum-downloads

### PR DESCRIPTION
**Pull request by orangemug**
## Description
Remove prerelease feature flags from curriculum-downloads (docx) and make it live 🎉

## Issue(s)

`CUR-757`

### CODE REVIEW

File: src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx [hunk 1]
```diff
@@ -77,16 +77,6 @@ describe("Component - Curriculum Header", () => {
...
-    expect(tabularNav).toBeInTheDocument();
-    const links = await findAllByRole(tabularNav, "link");
-    expect(links).toHaveLength(2);
-    expect(links[0]).toHaveTextContent("Unit sequence");
-    expect(links[1]).toHaveTextContent("Overview");
-  });
-
-  test("user can see the tabular navigation (downloads prerelease)", async () => {
     // NOTE: This is only active during testing.
     mockPrerelease("curriculum.downloads");
```
Comment (sonalisin) [inline src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx:81]: Do we still need to mock the pre-release?

File: src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx [hunk 2]
```diff
@@ -77,16 +77,6 @@ describe("Component - Curriculum Header", () => {
   });
 
   test("user can see the tabular navigation", async () => {
-    const { findByTestId } = renderComponent();
```
Comment (sonalisin) [inline src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx:80]: Should we also test to see we can see the downloads tab?

**Review by sonalisin — CHANGES_REQUESTED** [review]

File: src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx (hunk 2 above)
Comment (orangemug) [reply src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx:80]: We do test for that see line 91

```
expect(links[2]).toHaveTextContent("Download");
```

File: src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx (hunk 1 above)
Comment (orangemug) [reply src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx:81]: Yeah I think we can use it for now for the visualiser cycle 2 stuff, until we get proper flags implemented.

**Review by sonalisin — APPROVED** [review]

**Review by rightsaidjames — APPROVED** [review]
jazz_show_nice.gif

**Pull request merged**